mkt/websites/serializers.py

```diff
@@ -10,19 +10,18 @@
 class WebsiteSerializer(serializers.ModelSerializer):
     categories = ListField(serializers.CharField())
     description = TranslationSerializerField()
-    device_types = ListField(serializers.CharField(), source='device_names')
     id = serializers.IntegerField(source='pk')
     short_name = TranslationSerializerField()
     name = TranslationSerializerField()
     title = TranslationSerializerField()
     icons = serializers.SerializerMethodField('get_icons')
 
-    # FIXME: keywords, regions. try to stay compatible with Webapp API
-    # as much as possible.
+    # FIXME: keywords. need to refactor Webapp code. Probably want to use
+    # tag_list and associated transformer to keep things DRY and coherent.
```

Comment:
I started to implement keywords in this PR but stopped when I realized I needed to make sure blocked tags were not included too. Some refactoring is needed, I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1168817